ReactWindows/ReactNative/Modules/DeviceInfo/DeviceViewInfo.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Windows.Graphics.Display;
+using Windows.UI.ViewManagement;
+using ReactNative.Bridge;
+using ReactNative.UIManager;
+
```

Comment:
usings are alphabetical in this code base (no preference for System or Windows first)